src/services/user.js

```diff
@@ -9,13 +8,17 @@ import session from './session';
 import storage from './storage';
 
 /**
- * Get all activities for a session, sorted alphabetically by description
+ * Get events attended by the user
  * @param {String} sessionCode Identifier for a session
- * @return {Promise.<User[]>}
+ * @return {Promise.<attendedEvents[]>}
```

Comment:
Same as my comment above about `DaysLeft`, you have to define this type before you can use it in a documentation block. Note that the `[]` syntax means "an array of this type", so you would want the name of the type you use to be singular, like `AttendedEvent`.